### DISCUSSION TITLE
coinex to chia wallet

### DISCUSSION THREAD

**Discussion by Babiskixsi** [Chia-Network/chia-blockchain#8377] [Support]
Hello and have a good time

I transferred 1 unit of Chia from coinex site to the wallet inside the Chia version of Windows, but so far, despite the confirmation of the transaction, nothing has been deposited in the last 8 hours and my balance is still zero.
What is the problem now?

![011](https://user-images.githubusercontent.com/90283775/132404178-afc89809-073d-43ed-bd3f-d843939a06b7.JPG)

**Comment by jack60612** [comment]
Are you synced?

**Reply by Babiskixsi** [reply]
I saw a not synced message in the corner of the app but I thought it might be for farming, what should I do about it?

![012](https://user-images.githubusercontent.com/90283775/132409315-7e9a2397-2e1b-44ab-bc92-50b580ca3950.JPG)

**Reply by jack60612** [reply]
In order to send your coins you need to be synced. Once you are synced then you will be able to see the coins and spend the coins.

**Comment by BrandtH22** [comment]
@Babiskixsi , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
